doc/sphinx/docs/algorithm_list.rst

```diff
@@ -27,6 +27,7 @@ Heuristic Global Optimization
 Common Name                                                Docs of the C++ class                     Docs of the python class                  Capabilities
 ========================================================== ========================================= ========================================= =========================
 The null algorithm [#null_algo]_                           :cpp:class:`pagmo::null_algorithm`        :class:`pygmo.null_algorithm`             SM-CU-I-sto
+Ant Colony Optimization (ACO)                              :cpp:class:`pagmo::gaco`                  :class:`pygmo.gaco`                       S-CU-I
```

Comment:
ACO is reserved for other algorithms that are not this extended version. Use (extended ACO)